YUViewLib/ui/videoHandlerRGB.ui

```diff
@@ -154,28 +180,32 @@
          </widget>
         </item>
         <item row="4" column="0">
-         <widget class="QLabel" name="label_6">
+         <widget class="QLabel" name="labelScaleA">
           <property name="toolTip">
-           <string>Select this if the input values are using the limited MPEG range of values.</string>
-          </property>
-          <property name="whatsThis">
-           <string>Select this if the input values are using the limited MPEG range of values.</string>
+           <string>Scale every value in the alpha channel with this factor</string>
           </property>
           <property name="text">
-           <string>Limited Range</string>
+           <string>Scale A</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
-         <widget class="QCheckBox" name="limitedRangeCheckBox">
+         <widget class="QSpinBox" name="AScaleSpinBox">
           <property name="toolTip">
-           <string>Select this if the input values are using the limited MPEG range of values.</string>
+           <string>Scale every value in the alpha channel with this factor</string>
           </property>
-          <property name="whatsThis">
-           <string>Select this if the input values are using the limited MPEG range of values.</string>
+          <property name="value">
```

Comment:
Shouldn't this be `<property name="minimum">`?
I noticed this because from the UI I could scale the alpha to 0.

However, come to think of it, scaling to 0 could be a nice trick to turn off the individual components, and it would come in handy if you decided to apply the alpha component by default in the view, i.e. the user can scale alpha to 0 to turn it off in the rendered view. But this can be a follow up change of course.

Reply:
Yep I am not yet done with it. I just opened a PR already to test building. 
Good point. I was also thinking about the possibility to turn off the rendering of the alpha component later. Setting the scale to 0 may be one option. But then we should/could do the same for the other components. Another option would be to add this as a render option in the drop-down list (have one option RGB and another with RGBA). I think I prefer the second one actually.

Reply:
Turning off the components individually by scaling them to 0 might come in handy and it is allowed in the YUV case AFAICS, but either way is fine with me, thanks again.

Reply:
I could just enable both ways. Why not. More flexibility.